tico/serialize/operators/op_any.py

```diff
@@ -99,17 +100,10 @@ def define_node(
         keepdim = args.keepdim
 
         input_shape = list(extract_shape(input))
-        dim_i32 = None
         if dim is None:
-            dims = tuple(i for i in range(0, len(input_shape)))
-            dim_i32 = tuple(
-                circle_legalize_dtype_to(dim, dtype=torch.int32) for dim in dims
-            )
-        if isinstance(dim, int):
-            dim_i32 = circle_legalize_dtype_to(dim, dtype=torch.int32)
-        if isinstance(dim, tuple):
-            dim_i32 = tuple(circle_legalize_dtype_to(d, dtype=torch.int32) for d in dim)
```

Comment:
@mhs4670go Look at this usage of circle_legalize_dtype_to. I guess there's some bad usage of this too.